From: Sang-Heon Jeon <ekffu200098@gmail.com>
Date: Tue, 12 Aug 2025 23:00:46 +0900
Subject: [PATCH] selftests/damon: fix selftests by installing drgn related
 script

drgn_dump_damon_status is not installed while kselftest setup.
It can breaks other tests which depend on drgn_dump_damon_status.
Install drgn_dump_damon_status files to fix broken test.

Signed-off-by: Sang-Heon Jeon <ekffu200098@gmail.com>
Cc: Alexandre Ghiti <alexghiti@rivosinc.com>
Link: https://patch.msgid.link/20250812140046.660486-1-ekffu200098@gmail.com
Reviewed-by: SeongJae Park <sj@kernel.org>
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/testing/selftests/damon/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/selftests/damon/Makefile b/tools/testing/selftests/damon/Makefile
index fdb385cca748..3d8ba66e8e27 100644
--- a/tools/testing/selftests/damon/Makefile
+++ b/tools/testing/selftests/damon/Makefile
@@ -5,6 +5,7 @@ TEST_GEN_FILES += access_memory access_memory_even
 
 TEST_FILES = _damon_sysfs.py
 TEST_FILES += _common.sh
+TEST_FILES += drgn_dump_damon_status.py
 
 # functionality tests
 TEST_PROGS += sysfs.sh
-- 
2.39.5

